Applications/DataExplorer/DataView/DiagramView/DiagramScene.cpp

```diff
@@ -142,17 +142,17 @@ void DiagramScene::adjustAxis(float &min, float &max, int &numberOfTicks)
     numberOfTicks = int(ceil(max / step) - std::floor(min / step));
     if (numberOfTicks < MinTicks)
         numberOfTicks = MinTicks;
-    min = std::floor(min / step) * step;
-    max = ceil(max / step) * step;
+    min = (std::floor(min / step) * step);
+    max = (ceil(max / step) * step);
```

Comment:
`std::ceil`
outer parentheses not needed.

Reply:
I'm not sure how these got there. Is it possible that clang-format does something like that?

Anyway, I've removed them again... :heavy_check_mark:

Reply:
no, clang-format does not add parentheses in this case... It would not change the formatting at all, because it is already correct... ;)